code/__DEFINES/traits.dm

```diff
@@ -260,3 +260,4 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define MADE_UNCLONEABLE "made-uncloneable"
 #define TIMESTOP_TRAIT "timestop"
 #define STICKY_NODROP "sticky-nodrop" //sticky nodrop sounds like a bad soundcloud rapper's name
+#define GAMER_GOD "gamer-god" //double arcade prizes
```

Comment:
shouldn't this be TRAIT_GAMERGOD?

Reply:
on that note, all these traits fucked up there. if you could be so epic to fix the naming conventions on all of them thatd be epic sauce but dont feel like you have to

Reply:
these are trait sources, not traits, so GAMER_GOD_TRAIT

Even then, that convension is probably ambiguous when compared with the trait naming itself. Maybe TRAITSOURCE_[thing].

Reply:
changing all of them would touch a lot of files and be really out of scope, but i'll fix the one i've added

Reply:
>these are trait sources, not traits

really? i thought these were traits